pkgs/desktops/xfce/core/xfdesktop.nix

```diff
@@ -19,6 +19,13 @@ stdenv.mkDerivation rec {
       libglade xfce4panel thunar exo garcon libnotify hicolor_icon_theme
     ];
 
+  patches =
+    [
+      (fetchurl {
+      url = https://git.xfce.org/users/eric/xfdesktop/patch/?id=cc311b61b82b7510a3a6cb0952d3a331e3551e05;
```

Comment:
You can use directly https://git.xfce.org/xfce/xfdesktop/patch?id=157f5b55cfc3629d595ef38984278de5915aac27